strings-csharp/DeterminingIfStringIsNumericOnly/Tests/IsStringNumberUnitTests.cs

```diff
@@ -0,0 +1,171 @@
+using DeterminingIfStringIsNumericOnly;
+
+namespace Tests;
+
+public class IsStringNumberUnitTests
```

Comment:
Since with all but one exception (at least from a quick perusal) there are two constants being used for the test cases: a true case and a false case. These constants could be elevated to the class level and referenced there.
If we wanted to reduce the amount of code in the test class, we could go a step further and make use of the TestCaseSource attribute and add a simple static property that yielded the two strings along with the expected boolean outcome. Then we can just add two parameters to each of the tests (with the exception of the large integer for the int.TryParse case): string input and bool expectedResult

Reply:
Added TestCase attributes to tests using the static properties (except for the TryParse when the input is larger than int.MaxValue, there I just passed a long string to the TestCase attribute)